integration-cli/docker_api_info_test.go

```diff
@@ -64,13 +70,15 @@ func (s *DockerSuite) TestInfoAPIRuncCommit(c *check.C) {
 
 func (s *DockerSuite) TestInfoAPIVersioned(c *check.C) {
 	testRequires(c, DaemonIsLinux) // Windows only supports 1.25 or later
-	endpoint := "/v1.20/info"
 
-	status, body, err := request.SockRequest("GET", endpoint, nil, daemonHost())
-	c.Assert(status, checker.Equals, http.StatusOK)
+	res, body, err := request.Get("/v1.20/info")
```

Comment:
`ExecutionDriver` isn't in `types.Info` struct so we can't check if it exists if we will use `cli.Info`

https://github.com/sbko/moby/blob/29276d647f5c4e0a1e9fafd62923872292d01fe2/integration-cli/docker_api_info_test.go#L82